app/scripts/modules/mhd-doc-builder.js

```diff
@@ -73,26 +73,26 @@ exports.createDocumentBundle = (patientRef, eventDictionaries, currentTime) => {
         value: uuid.v4()
       },
       resourceType: 'Composition',
-      status: 'final',
+      status: 'preliminary',
       type: {
         coding: {
-          system: 'http://hl7.org/fhir/ValueSet/c80-doc-typecodes',
-          code: '74264-3'
+          system: 'http://opencrvs.org/doc-types',
+          code: 'birth-notification'
```

Comment:
Im wondering if these should be more generic to cater for all events being added, Or we will need to have a few variable placeholders depending on the event being submitted
- birth-notification
- immunization